events/_event-post-macros.html

```diff
@@ -81,19 +80,25 @@
 
    Create HTML event meta markup when given:
 
-   post: A post from a query result.
+   post:                        A post from a query result.
+
+   address_format (optional):   A python string format to display to 
+                                address information  
+                                (e.g. '{venue} - {city}, {state}').
+
+   datetime_format (optional):  An object to use for customizing event-meta 
+                                time and date formats.
 
    ========================================================================== #}
 
-{% macro event_meta(post) %}
+{% macro event_meta(post, address_format='{city} {state}', datetime_format={})%}
```

Comment:
No, you need those if you are using named arguments.

https://docs.python.org/2/library/string.html#format-string-syntax